iroha_data_model/src/lib.rs

```diff
@@ -969,6 +1013,22 @@ pub mod asset {
         pub const fn new_store(id: DefinitionId) -> Self {
             AssetDefinition::new(id, AssetValueType::Store)
         }
+
+        /// Checks the length of the id is in a valid range
+        ///
+        /// # Errors
+        /// Fails if limit check fails
+        pub fn validate_len(&self, limits: LengthLimits) -> Result<()> {
```

Comment:
Api can be changed to
```rust
fn validate_len(&self, limits: Into<LengthLimits>) -> Result<()>;
```

In such way it would be more abstract

Reply:
Preferred varian below with `Into<RangeInclusive<usize>>`

Reply:
Yeah, that's what i meant 😅